endpoints/openrtb2/video_auction.go

```diff
@@ -242,14 +242,19 @@ func cleanupVideoBidRequest(videoReq *openrtb_ext.BidRequestVideo, podErrors []P
 
 func handleError(labels pbsmetrics.Labels, w http.ResponseWriter, errL []error, ao analytics.AuctionObject) {
 	labels.RequestStatus = pbsmetrics.RequestStatusErr
-	w.WriteHeader(http.StatusInternalServerError)
+	if errortypes.DecodeError(errL[0]) == errortypes.BlacklistedAppCode {
```

Comment:
Again, we should check the entire `errs` slice rather than just the first element.

Reply:
corrected